db_stress_tool/cf_consistency_stress.cc

```diff
@@ -486,6 +489,39 @@ class CfConsistencyStressTest : public StressTest {
     } while (true);
   }
 
+  void ContinuouslyVerifyDb(ThreadState* thread) const override {
+    assert(cmp_db_ != nullptr);
+    Status s = cmp_db_->TryCatchUpWithPrimary();
+    ReadOptions ropts;
+    ropts.total_order_seek = true;
+    uint32_t crc = 0;
+    {
+      // Compute crc for all key-values of default column family.
+      std::unique_ptr<Iterator> iter(cmp_db_->NewIterator(ropts));
+      for (iter->SeekToFirst(); iter->Valid(); iter->Next()) {
+        crc = crc32c::Extend(crc, iter->key().data(), iter->key().size());
+        crc = crc32c::Extend(crc, iter->value().data(), iter->value().size());
+      }
```

Comment:
This feels duplicated code as line 510 to 516 to me. Can we consolidate the code?

Reply:
Sure.